docker/nginx/nginx.conf

```diff
@@ -29,4 +29,52 @@ http {
             proxy_http_version 1.1;
         }
     }
+
+    server {
+        listen 8081 ssl;
+        ssl_certificate /etc/ssl/certs/testCert.crt;
+        ssl_certificate_key /etc/ssl/private/testCert.rsa;
+
+        server_name _;
+
+        access_log off;
+        error_log stderr warn;
+
+        location / {
+            proxy_pass http://backend;
+            proxy_http_version 1.1;
+        }
+    }
+
+    server {
+        listen 8082 ssl;
+        ssl_certificate /etc/ssl/certs/testCert.crt;
+        ssl_certificate_key /etc/ssl/private/testCert.rsa;
+
+        server_name _;
+
+        access_log off;
+        error_log stderr warn;
+
+        location / {
+            proxy_pass https://backend;
+            proxy_http_version 1.1;
+        }
+    }
+
+    server {
+        listen 8083 http2 ssl;
+        ssl_certificate /etc/ssl/certs/testCert.crt;
+        ssl_certificate_key /etc/ssl/private/testCert.rsa;
+
+        server_name _;
+
+        access_log off;
+        error_log stderr warn;
+
+        location / {
+            proxy_pass https://backend;
```

Comment:
The scenario doesn't dictate if the destination should use http or https. Why did you pick https?

Reply:
It's HTTPS-HTTPS scenario. Or is it redundant?

Reply:
[This one is HTTPS-HTTP](https://github.com/aspnet/Benchmarks/blob/7c7d00e2652a50bbc514ed1c9b9a423731d95b04/docker/nginx/nginx.conf#L34).

Reply:
This is the http2-http1 scenario. The destination could use either http or https. I just want to know why you picked https, it seems like unneeded overhead.

Reply:
Ah, sorry, I misunderstood. Yes, you are probably right. I will change it back to http.

Reply:
Done